src/_includes/partials/components/checkbox.macro.njk

```diff
@@ -0,0 +1,6 @@
+{%- macro checkbox(params) -%}
+    <div>
+        <input class="checkbox" type="checkbox" id="{{ params.id }}" name="{{ params.name }}">
+        <label for="{{ params.id }}">{{ params.label }}</label>
+    </div>
+{% endmacro %}
```

Comment:
I don't think we should have a macro for this. There's no way to pass the `checked` attribute right now, we might not want checkboxes wrapped in a `<div>`.

Realistically we will be outputting these with a loop like so:

```njk
{% for topic in topics %}
{% set checked = selectedTopics.includes(topic.uuid) %}
<li>
  <input type="checkbox" id="{{ topic.title | slugify }}" name="{{ topic.uuid }}" {% if checked  %}checked{% endif -%} />
  <label for="{{ topic.uuid }}">{{ topic.title }}</label>
</li>
```

In my opinion a macro would kind of be overengineering this.